test/utils/block.ts

```diff
@@ -556,7 +556,7 @@ export const findEraBlockUsingBinarySearch = async (
     let currentEraIndex = 0;
 
     const runtimeUpgradedToSupportEraAt = await getBlockNumberAtWhichEraStarted(api);
-    console.log("Runtime upgrade to support era at:", runtimeUpgradedToSupportEraAt);
+    // console.log("Runtime upgrade to support era at:", runtimeUpgradedToSupportEraAt);
```

Comment:
why removing this¿

Reply:
Reverted, was annoying log during debug.